game/templates/game/landing.html

```
@@ -1123,6 +1123,140 @@ <h2 style="color:#fff; font-size:1.05rem; font-weight:600; margin-bottom:8px;">
   })();
   </script>
+  <!-- Keyboard Shortcuts Modal -->
+<div id="shortcutsModal"
+     style="display:none;position:fixed;inset:0;background:rgba(0,0,0,.85);z-index:99999;align-items:center;justify-content:center;">
+
+  <div style="background:#131522;color:white;padding:24px;border-radius:16px;width:90%;max-width:600px;border:1px solid rgba(240,192,64,.3);">
+
+    <div style="display:flex;justify-content:space-between;align-items:center;margin-bottom:20px;">
+      <h2 style="color:#f0c040;">Keyboard Shortcuts</h2>
+
+      <button onclick="closeShortcutsModal()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)